config/middlewares.js

```diff
@@ -78,11 +78,21 @@ module.exports = [
       sameSite: process.env.NODE_ENV === "development" ? 'Lax' : 'None',
     },
   },
+
   'strapi::poweredBy',
   'strapi::logger',
   'strapi::query',
-  'strapi::body',
   'strapi::favicon',
   'strapi::public',
   'plugin::users-permissions.jwt',
+  {
```

Comment:
Looks like there is a conflict here because we recently added https://github.com/dev-launchers/strapiv4/blob/main/config/middlewares.js#L88-L94. W should keep that and the body middleware that you are adding.